ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,424 @@
+
+### Physical L2LS Topology
+
+The drawing below shows the physical topology used in this example. The interface assignment shown here are referenced across the entire example, so keep that in mind if this example must be adapted to a different topology.
+
+![Figure: 1](images/pure_L2LS_topo.svg)
+
+???+ note
+
+    The FW/L3 Device and individual hosts (A-D) in this example are not managed by AVD, but the switch ports connecting to these devices are.
+
+## Basic EOS Switch Configuration
+
+Basic connectivity between the Ansible controller host and the switches must be established before Ansible can be used to deploy configurations. The following should be configured on all switches:
+
+- Switch Hostname
+- IP enabled interface
+- Username and Password defined
+- Management eAPI enabled
+
+???+ info
+
+    When using vEOS/cEOS virtual switches, `Management0` is used. When using actual hardware switches, `Management1` is used. The included basic switch configurations may need to be adjusted for your environment.
+
+Below is the basic configuration file for SPINE1:
+
+```shell
+--8<--
+examples/l2ls-fabric/switch-basic-configurations/SPINE1.cfg
+--8<--
+```
+
+## Ansible Inventory
+
+Now that we understand the physical L2LS topology, we need to create the Ansible inventory that represents this topology. The following is a textual and graphical representation of the Ansible inventory group variables and naming scheme used in this example:
+
+``` text
+- DC1
+  - DC1_FABRIC
+    - DC1_SPINES
+    - DC1_LEAFS
+      - POD1
+      - POD2
+  - DC1_NETWORK_SERVICES
+    - DC1_SPINES
+    - DC1_LEAFS
+  - DC1_NETWORK_PORTS
+    - DC1_SPINES
+    - DC1_LEAFS
+```
+
+DC1 represents the highest level within the hierarchy. Ansible variables defined at this level will be applied to all nodes in the fabric. Ansible groups have parent and child relationships. For example, both DC1_SPINES and DC1_LEAFS are children of DC1_FABRIC. Groups of Groups are possible and allow variables to be shared at any level within the hierarchy. For example, DC1_NETWORK_SERVICES is a group with two other groups defined as children: DC1_SPINES and DC1_LEAFS. The same applies to the group named DC1_NETWORK_PORTS. You will see these groups listed at the bottom of the inventory file.
+
+This naming convention makes it possible to extend anything quickly but can be changed based on your preferences. The names of all groups and hosts must be unique.
+
+![Figure: 2](images/ansible_groups.svg)
+
+### inventory.yml
+
+The below inventory file represents 2 Spines and 2 Leaf Pairs (POD1 & POD2). The nodes are defined under the groups DC1_SPINES and DC1_LEAFS, respectively. We apply group variables (group_vars) to these groups to define their functionality and configurations.
+
+???+ note
+
+    The Leaf nodes belong to subgroups POD1 and POD2. Leaf nodes defined as a pair in this manner will have MLAG configured between them. In addition, you can filter the VLANs that are applied to the POD.
```

Comment:
Not true. We do _not_ require inventory groups to be per pair. I usually just put all leaf switches in a single group. The MLAG pairing only happens under the `l3leaf` config.

Reply:
removed this note...